src/client/shadows/shadowsshadercallbacks.h

```diff
@@ -42,20 +42,19 @@ class ShadowUniformSetter : public IShaderUniformSetter
 class ShadowUniformSetterFactory : public IShaderUniformSetterFactory
 {
 public:
-	virtual IShaderUniformSetter *create() {
+	virtual IShaderUniformSetter *create(const std::string &name) {
+		if (str_starts_with(name, "shadow/"))
```

Comment:
I realize that is what we had all the discussions about in the original PR... That said, isn't it a bit clumsy to compare the shader path here?

Reply:
Yes :shrug: